example/generated-src/objc/TXSItemList.mm

```diff
@@ -21,7 +21,14 @@ + (nonnull instancetype)itemListWithItems:(nonnull NSArray<NSString *> *)items
 
 - (NSString *)description
 {
-    return [NSString stringWithFormat:@"<%@ %p items:%@>", self.class, self, self.items];
+    return [NSString stringWithFormat:@"<class, %@ : %p, dict, %@>", self.class, self, [[self toDict] description]];
+}
+
+- (NSDictionary *)toDict
+{
+    #define _djinni_hide_null_(_o_) ((_o_)?(_o_):([NSNull null]))
+    
+    return @{@"__class_name__": [self.class description], @"items": _djinni_hide_null_(self.items)};
```

Comment:
Is the use of __class_name__ here a standard thing in ObjC for any other specific format, or something made up for Djinni?